Tests/SentryTests/Networking/RateLimits/SentryRateLimitCategoryMapperTests.swift

```diff
@@ -2,17 +2,37 @@ import XCTest
 @testable import Sentry
 
 class SentryRateLimitCategoryMapperTests: XCTestCase {
+    
+    private let categoryError = "error"
```

Comment:
l: should we make `SentryRateLimitCategoryError` visible only for tests?

Reply:
I also always used constants in my tests. I like to define my own constants in my tests, because if somebody accidentally modifies the test are green and you break the code. If you have it duplicated you have to make the error twice.